tagstudio/src/qt/widgets/thumb_renderer.py

```diff
@@ -112,156 +116,172 @@ def render(
                 ),
                 math.floor(12 * ThumbRenderer.font_pixel_ratio),
             )
-
-        if isLoading or filepath:
+        if ThumbRenderer.settings.value("render_thumbnails", False, bool):
```

Comment:
this variable is called `render_thumbnails` but when it's true it doesnt render thumnails. So either this condition should be flipped, or the variable name should be changed.